version/version.go

```diff
@@ -14,13 +14,13 @@ var (
 
 // MaxOpVersion and APIVersion supported
 const (
-	MaxOpVersion = 40100
+	MaxOpVersion = 50000
 	APIVersion   = 1
 )
 
 // GlusterdVersion and GitSHA
 var (
-	GlusterdVersion = "4.1.0"
+	GlusterdVersion = "5.0.0-rc0"
```

Comment:
When I had introduced this variable, the intention was to not initialise it with any hard-coded value. We populate it during build time:

**scripts/build.sh**
```sh
LDFLAGS="-X ${REPO_PATH}/version.GlusterdVersion=${VERSION} -X ${REPO_PATH}/version.GitSHA=${GIT_SHA}"
```

Reply:
I understand that this is always reset by the build script. But I didn't want to leave it at 4.1.0.

Reply:
Can be set to an empty string.